### PR TITLE
populate with drought duration

### PR DESCRIPTION
**Pull request by ceenell**
This is adding the horizontal scroll vue page. I'm committing two timeline charts to get the page working to test locally. The next steps will be for @hcorson-dosch  to add the relevant pipeline steps from `what-is-drought` so these are generated here. Please review to ensure the page runs locally given the readme description.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+VUE_APP_TITLE='drought-duration'
```
Comment (hcorson-dosch) [inline .env:1]:
```suggestion
VUE_APP_TITLE='drought-timeline'
```

File: .env [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+VUE_APP_FOOTER_1='https://labs.waterdata.usgs.gov/visualizations/temperature-prediction/index.html#/'
+VUE_APP_FOOTER_2='https://labs.waterdata.usgs.gov/visualizations/snow-to-flow/index.html#/'
+VUE_APP_GITHUB_REPOSITORY_LINK='https://github.com/USGS-VIZLAB/drought-duration'
```
Comment (hcorson-dosch) [inline .env:5]:
```suggestion
VUE_APP_GITHUB_REPOSITORY_LINK='https://github.com/USGS-VIZLAB/drought-timeline'
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "drought-duration",
```
Comment (hcorson-dosch) [inline package.json:2]:
```suggestion
  "name": "drought-timeline",
```

File: src/assets/text/droughtEvents.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
+export default {
```
Comment (hcorson-dosch) [inline src/assets/text/droughtEvents.js:1]: Nice to have this already partly populated!

File: jenkins/Jenkinsfile.build [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+        }
+        stage('checkout'){
+            steps{
+                checkout([$class: 'GitSCM',
+                          branches: [[name: "${params.BRANCH_TAG}"]],
+                          doGenerateSubmoduleConfigurations: false,
+                          extensions: [],
+                          gitTool: 'Default',
+                          submoduleCfg: [],
+                          userRemoteConfigs: [[url: 'https://github.com/usgs-vizlab/drought-duration.git']]
```
Comment (hcorson-dosch) [inline jenkins/Jenkinsfile.build:31]:
```suggestion
                          userRemoteConfigs: [[url: 'https://github.com/usgs-vizlab/drought-timeline.git']]
```

File: jenkins/Jenkinsfile.build [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+                        ])
+                }
+        }
+
+        stage('build') {
+            steps {
+
+                sh """
+                    docker build . --tag="drought-duration-docker" --build-arg BUILDTARGET=${params.BUILD_DEST} --build-arg VUE_BUILD_MODE=${params.VUE_BUILD_MODE}
```
Comment (hcorson-dosch) [inline jenkins/Jenkinsfile.build:40]:
```suggestion
                    docker build . --tag="drought-timeline-docker" --build-arg BUILDTARGET=${params.BUILD_DEST} --build-arg VUE_BUILD_MODE=${params.VUE_BUILD_MODE}
```

File: jenkins/Jenkinsfile.build [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+                    docker run drought-duration-docker
```
Comment (hcorson-dosch) [inline jenkins/Jenkinsfile.build:41]:
```suggestion
                    docker run drought-timeline-docker
```

File: jenkins/Jenkinsfile.build [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+                    pathtemplate=":tmp/drought-duration/dist"
```
Comment (hcorson-dosch) [inline jenkins/Jenkinsfile.build:42]:
```suggestion
                    pathtemplate=":tmp/drought-timeline/dist"
```

File: jenkins/Jenkinsfile.build [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+                """
+
+            }
+        }
+
+        stage('send to S3') {
+            steps {
+                script {
+                    if ("${params.BUILD_DEST}" == "prod") {
+                        targetDomain = "s3://water-visualizations-prod-website/visualizations/drought-duration"
```
Comment (hcorson-dosch) [inline jenkins/Jenkinsfile.build:55]:
```suggestion
                        targetDomain = "s3://water-visualizations-prod-website/visualizations/drought-timeline"
```

File: jenkins/Jenkinsfile.build [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+                    }
+                    else if ("${params.BUILD_DEST}" == "beta") {
+                        targetDomain = "s3://water-visualizations-beta-website/visualizations/drought-duration"
```
Comment (hcorson-dosch) [inline jenkins/Jenkinsfile.build:58]:
```suggestion
                        targetDomain = "s3://water-visualizations-beta-website/visualizations/drought-timeline"
```

File: jenkins/Jenkinsfile.build [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+                    }
+                    else {
+                        targetDomain = "s3://water-visualizations-test-website/visualizations/drought-duration"
```
Comment (hcorson-dosch) [inline jenkins/Jenkinsfile.build:61]:
```suggestion
                        targetDomain = "s3://water-visualizations-test-website/visualizations/drought-timeline"
```

File: public/index.html [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+      <meta property="twitter:card" content="summary_large_image">
+      <meta property="twitter:url" content="https://labs.waterdata.usgs.gov/visualizations/gw-conditions/index.html#/">
+      <meta property="twitter:title" content="70 years of hydrological drought">
+      <meta property="twitter:description" content="">
+      <meta property="twitter:image" content="https://labs.waterdata.usgs.gov/visualizations/thumbnails/gw-conditions-og-peaks.png">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+<script type='application/ld+json'>
+      { "@context": "http://www.schema.org",
+        "@type": "WebSite", "name": "70 years of hydrological drought",
+        "url": "https://labs.waterdata.usgs.gov/visualizations/drought-duration/index.html#/",
```
Comment (hcorson-dosch) [inline public/index.html:35]:
```suggestion
        "url": "https://labs.waterdata.usgs.gov/visualizations/drought-timeline/index.html#/",
```

**Review by hcorson-dosch — APPROVED** [review]
Looks good, Cee! This built without issue for me. The horizontal scroll is really neat -- nice work figuring that out.

The suggested changes that I added are just to add the disclaimer to the README and to switch any `drought-duration` to `drought-timeline` - ignore any of those if they mess up the build.

**Comment by ceenell** [comment]
thanks for checking and switching all the duration to timeline, I was in a rush to put this up and hadn't addressed that

**Pull request merged**